app/js/controllers/resultat.js

```diff
@@ -46,7 +46,7 @@ angular.module('ddsApp').controller('ResultatCtrl', function($scope, $rootScope,
     };
 
     $scope.shouldDisplayImpactInfo = function(partenaireId) {
-        return ! droitsDescription.partenairesLocaux[partenaireId].interactionWithNationalPrestationCalculated;
+        return droitsDescription.partenairesLocaux[partenaireId].interactionWithNationalPrestationCalculated !== false;
```

Comment:
Ma compréhension de la situation, il y a un opt-in pour supprimer l'avertissement. Dit autrement, l'avertissement est affiché par défaut et il peut être supprimé avec un paramètre supplémentaire correctement renseigné.

Avec la modification, il s'agit toujours d'un opt-in pour supprimer l'avertissement. Je trouvais que la convention de nommage n'était pas claire. Ma modification en l'état ne clarifie pas non plus.

Je modifie le _flag_ en `shouldDisplayPotentialInteractionWarning' ?